index.js

```diff
@@ -205,7 +208,7 @@ module.exports = function (opts) {
         filename: false,
         style: 'solarized-dark',
         template: ROOT+'src/layout.html',
-        scss: ROOT+'src/app.scss',
```

Comment:
you mean the current version does not work in one case? can you explain further?